packages/yoastseo/spec/scoring/assessments/seo/ProductSKUAssessmentSpec.js

```diff
@@ -150,3 +157,61 @@ describe( "a test for the applicability of the assessment", function() {
 		expect( isApplicable ).toBe( true );
 	} );
 } );
+
+
+describe( "test the applicabilityHelper", () => {
```

Comment:
Same as above, this can be moved to "a test for the applicability of the assessment"

Reply:
Good idea!